CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Deleted x-google-marketplace section for tester.image in schema.yaml to be consistent for Google Marketplace Tools v0.10.0
 - Added a build.sh flag (`-p` or `--persist`) to persist the application deployment after testing.
 - Fixed the deployment's `deploy-info` annotations to use keys that are valid JSON keys (i.e. with quotes).
+- Adds instructions for building, testing, releasing, and publishing to CONTRIBUTING.md
+- Adds a mechanism for contributors outside of CyberArk to build and test new versions of the Marketplace without requiring access to the conjur-cloud-launcher-onboard GCP project.
```

Comment:
The two lines here are probably unnecessary in the changelog as they do not impact the end-user (installation) usage of the repo nor the release.